packages/opc-menu-drawer/README.md

```diff
@@ -178,17 +187,68 @@ document.querySelector("opc-menu-drawer").links = {
   <opc-menu-drawer headerTitle="Akhil Mohan"></opc-menu-drawer isOpen>
 ```
 
+### Methods
+
 - `isOpen`
   - Type: `Boolean`
   - Default value: `false`
 
-```html
-<opc-menu-drawer isOpen></opc-menu-drawer>
+```js
+document.querySelector('opc-menu-drawer').isOpen;
+```
+
+- `open`
+  - Description: To open the drawer.
+
+```js
+document.querySelector('opc-menu-drawer').open();
+```
+
+- `close`
+  - Description: To close the drawer.
+
+```js
+document.querySelector('opc-menu-drawer').close();
+```
+
+- `toggle`
+  - Description: To toggle the drawer states.
+
+```js
+document.querySelector('opc-menu-drawer').toggle();
 ```
```

Comment:
Why I didn't add was its not an attribute. So in control knob adding them would be difficult and without that it would be meaningless right.
I may pull this issue in changing the storybook integration.